<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SearchVariable</class>
 <widget class="QDialog" name="SearchVariable">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Search Variable</string>
  </property>
  <property name="windowIcon">
   <iconset resource="qtc.qrc">
    <normaloff>:/cteicons/img/Search.ico</normaloff>:/cteicons/img/Search.ico</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QFrame" name="fraSearch">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="0" colspan="2">
       <widget class="QLabel" name="lblSections">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>12</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>8</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Select Section:</string>
        </property>
        <property name="buddy">
         <cstring>cboSections</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="5">
       <widget class="QLineEdit" name="txtNode">
        <property name="minimumSize">
         <size>
          <width>40</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>90</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="maxLength">
         <number>6</number>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QComboBox" name="cboPriority">
        <property name="minimumSize">
         <size>
          <width>70</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>70</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QComboBox" name="cboType">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
       </widget>
      </item>
      <item row="2" column="4">
       <widget class="QComboBox" name="cboProtocol">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="lblPriority">
        <property name="minimumSize">
         <size>
          <width>70</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>70</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>8</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Priority</string>
        </property>
        <property name="buddy">
         <cstring>cboPriority</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QLabel" name="lblType">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>8</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Type</string>
        </property>
        <property name="buddy">
         <cstring>cboType</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="5">
       <widget class="QLabel" name="lblNode">
        <property name="minimumSize">
         <size>
          <width>40</width>
          <height>0</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>8</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Node ID</string>
        </property>
        <property name="buddy">
         <cstring>txtNode</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QLabel" name="lblProtocol">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>8</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Protocol</string>
        </property>
        <property name="buddy">
         <cstring>cboProtocol</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QComboBox" name="cboSections">
        <property name="minimumSize">
         <size>
          <width>180</width>
          <height>22</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="toolTip">
         <string>Select Crosstable Area</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="lblVarname">
        <property name="text">
         <string>Search:</string>
        </property>
        <property name="buddy">
         <cstring>txtVarname</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="4">
       <widget class="QLineEdit" name="txtVarname"/>
      </item>
      <item row="1" column="6">
       <widget class="QLabel" name="lblPromptFound">
        <property name="text">
         <string>Found:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="5" colspan="2">
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>2</number>
        </property>
        <item>
         <widget class="QCheckBox" name="chkCase">
          <property name="minimumSize">
           <size>
            <width>90</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Case sensitive</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="chkStartsWith">
          <property name="text">
           <string>Name Starts with</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="6">
       <widget class="QLabel" name="lblFound">
        <property name="minimumSize">
         <size>
          <width>70</width>
          <height>0</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <property name="text">
         <string>0</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="tblVariables">
     <property name="font">
      <font>
       <family>MS Shell Dlg 2</family>
       <pointsize>8</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="styleSheet">
      <string notr="true">QDialogButtonBox
{
	dialogbuttonbox-buttons-have-icons: 1;
	icon-size: 32px 32px;
	dialog-apply-icon: url(:/cteicons/img/Done_32.png);
	dialog-cancel-icon: url(:/cteicons/img/Cancel_32.png);
	dialog-close-icon: url(:/cteicons/img/Close_32.png);
	dialog-discard-icon: url(:/cteicons/img/Undo.png);
	dialog-no-icon: url(:/cteicons/img/Cancel_32.png);
	dialog-ok-icon: url(:/cteicons/img/Done_32.png);
	dialog-open-icon: url(:/cteicons/img/Open_32.png);
	dialog-reset-icon: url(:/cteicons/img/Reset_32.png);
	dialog-save-icon: url(:/cteicons/img/Save.png);
	dialog-yes-icon: url(:/cteicons/img/Done_32.png);
}
</string>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok|QDialogButtonBox::Reset</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>txtVarname</tabstop>
  <tabstop>chkCase</tabstop>
  <tabstop>chkStartsWith</tabstop>
  <tabstop>cboSections</tabstop>
  <tabstop>cboPriority</tabstop>
  <tabstop>cboType</tabstop>
  <tabstop>cboProtocol</tabstop>
  <tabstop>txtNode</tabstop>
  <tabstop>tblVariables</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources>
  <include location="qtc.qrc"/>
 </resources>
 <connections/>
</ui>
